openapi.yaml

```diff
@@ -17,1237 +21,222 @@ produces:
-  MealPlansResultObject:
-    properties:
-      links:
-        $ref: '#/definitions/SelfLink'
-      data:
-        type: array
-        items:
-          $ref: '#/definitions/MealPlanResourceObject'
-  MealPlanResultObject:
-    properties:
-      links:
-        $ref: '#/definitions/SelfLink'
-      data:
-        $ref: '#/definitions/MealPlanResourceObject'
-  MealPlanResourceObject:
+  PersonResource:
+    type: object
     properties:
       id:
         type: string
-        description: Meal plan ID
+        description: OSU ID of a person
+        pattern: '^\d{9}$'
       type:
-        type: string
-        example: meal-plans
+        $ref: '#/definitions/PersonType'
       links:
-          $ref: '#/definitions/SelfLink'
+        $ref: '#/definitions/SelfLink'
       attributes:
+        type: object
         properties:
-          mealPlan:
+          firstName: &firstName
             type: string
-            description: Description of meal plan
-            example: Orange Rewards
-          balance:
-            type: number
-            format: float
-            description: Remaining balance on meal plan in USD
-            example: 23.8
-          lastUsedDate:
+            description: First name of person
+          lastName: &lastName
             type: string
-            format: date-time
-            description: The last time the meal plan was used
-            example: '2017-07-14T16:45:15Z'
-          lastUsedLocation:
+            description: Last name of person
+          middleName: &middleName
             type: string
-            description: Description of place the meal plan was last used
-            example: 'Java II'
-  NewAddressResultObject:
-    properties:
-      data:
-        $ref: '#/definitions/NewAddressResourceObject'
-  NewAddressResourceObject:
-    properties:
-      type:
-        $ref: '#/definitions/AddressType'
-      attributes:
-        required:
-        - addressType
-        - city
-        properties: &commonAddressAttributes
-          addressType:
+            description: Middle name of person
+          displayFirstName:
             type: string
-            description: Code for the type of address record.
-            example: CM
-          addressLine1:
+            description: Preferred first name
+          displayMiddleName:
             type: string
-            example: '123 Main Street'
-          addressLine2:
+            description: Preferred middle name
+          displayLastName:
             type: string
-            example: 'Apartment 123'
-          addressLine3:
+            description: Preferred last name
+          onid:
             type: string
-          addressLine4:
+            description: ONID username
+          email:
             type: string
-          houseNumber:
+            description: Primary email address
+          birthDate: &birthDay
             type: string
-          city:
+            description: Birthday of person
```

Comment:
Can you also add the format in description and an example? https://github.com/osu-mist/students-api/blob/9ab6cf63b26afc7802354ea4ef92ae56a12c1220/openapi.yaml#L735-L738